packages/core/src/blitz-app-root.tsx

```diff
@@ -38,19 +38,11 @@ const NoPageFlicker = () => {
   )
 }
 
-export function withBlitzInnerWrapper(Page: BlitzPage) {
-  const BlitzInnerRoot = (props: ComponentPropsWithoutRef<BlitzPage>) => {
-    useAuthorizeIf(Page.authenticate === true)
-
-    return <Page {...props} />
-  }
-  for (let [key, value] of Object.entries(Page)) {
-    ;(BlitzInnerRoot as any)[key] = value
-  }
```

Comment:
This is needed to to copy all values like `Page.authenticate` and `Page.getLayout` so that `Component` inside user's `_app` will have those values.